src/components/tile-manager/tile.ts

```diff
@@ -103,14 +105,29 @@ export default class IgcTileComponent extends EventEmitterMixin<
    * @attr
    */
   @property({ attribute: 'disable-drag', type: Boolean, reflect: true })
```

Comment:
Lit docs suggest putting the property decorator over the setter when implementing custom property accessors.